force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -5844,6 +5844,13 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>Button to process a batch and payments through {0}</shortDescription>
         <value>Process Batch &amp; Payments</value>
     </labels>
+    <labels>
+        <fullName>bgeEditPaymentInformation</fullName>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Warns users to re-enter payment information when editing Elevate-connected gifts</shortDescription>
+        <value>You can’t edit this Elevate-connected gift until you re-enter the payment information. Select Edit Payment Information.</value>
```

Comment:
Nitpick, but I like to avoid apostrophes in Custom Labels (especially this weird rich-text version!)

Reply:
updated